conformance/tests/directives_version_platform.py

```diff
@@ -0,0 +1,45 @@
+"""
+Tests checking the version or platform.
+"""
+
+# Specification: https://typing.readthedocs.io/en/latest/spec/directives.html#version-and-platform-checking
+
+import os
+import sys
+
+
+if sys.version_info >= (3, 8):
+    pass
+else:
+    val1: int = ""  # Should not generate an error
+
+if sys.version_info >= (3, 8, 0):
+    pass
+else:
+    val2: int = ""  # Should not generate an error
+
+if sys.version_info < (3, 8):
+    val3: int = ""  # Should not generate an error
+
+if sys.version_info < (3, 100, 0):
+    pass
+else:
+    val3: int = ""  # Should not generate an error
+
+
+if sys.platform == "bogus_platform":
+    val5: int = ""  # Should not generate an error
+
+if sys.platform != "bogus_platform":
+    pass
+else:
+    val6: int = ""  # Should not generate an error
+
+
+if os.name == "bogus_os":
```

Comment:
We should be more explicit in the spec here, e.g. mypy currently doesn't understand checks against `os.name`

Reply:
Yes, I agree. I have this on my list of items to discuss. I marked mypy as "Pass" since the spec is ambiguous here.